src/app.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+
+const localCurrency = (currencies) => {
+    let local = "";
+    for (let i = 0; i < currencies.length; i++) {
+        if (i>0) {
+            local += ' and ' + currencies[i].name + 's.';
+        }
+        else {
+            local= currencies[i].name;
+        }
+    }
+    console.log(local);
+    return local;
+}
+
+async function fetchCountryInformation() {
+    const BASE = "https://restcountries.com/";
+    const EndPoint = "v2/name/";
+    const searchBar = document.getElementById("search-bar");
+    const searchBarValue = searchBar.value;
+    searchBar.value = '';
+    // console.log('Printing the URL: ' + BASE + EndPoint + searchBarValue);
+    try {
+        const response = await axios.get(BASE + EndPoint + searchBarValue);
+        // console.log(response);
+        let countries = response.data;
+        // console.log(countries);
+        const listOFCountries = document.getElementById("listOfCountries")
+                countries
+            .map((country) =>
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten.
```suggestion
            .map(({ name, flag, population, currencies, capital, region }) =>
```